ons_alpha/bulletins/forms.py

```diff
@@ -2,6 +2,9 @@
 
 
 class BulletinPageAdminForm(WagtailAdminPageForm):
+    class Meta:
```

Comment:
Issue: This isn't needed anymore.

Reply:
Thanks for the feedback. I've updated the BulletinPageAdminForm to remove the unnecessary Meta class.